### ISSUE TITLE
Support for Gnome 3.32

### ISSUE THREAD

**Issue by smecsia** [zakkak/workspace-grid#119]
Extension breaks with Gnome 3.32, could you please add support for it? It affects my daily work life heavily :)

**Comment by tom-james-watson** [comment]
https://extensions.gnome.org/extension/1485/workspace-matrix/ is a great alternative which supports 3.32

**Comment by zakkak** [comment]
Hello all,

I am no longer able to maintain this repository so if there is anyone willing to take over I would be glad to hand it over.

**Comment by zakkak** [comment]
Please see #120

**Comment by brianjmurrell** [comment]
Not just the extension breaks with Gnome 3.32 but it actually crashes gnome-shell.

**Comment by mikehardy** [comment]
@brianjmurrell at this point you should really transition to workspace-grid - the community has come together around that one for the future, and it works on 3.32

**Comment by brianjmurrell** [comment]
@mikehardy This is workspace-grid.  Do you mean [workspace-matrix](https://github.com/mzur/gnome-shell-wsmatrix)?

In any case, I already have switched but miss #67 as a feature of workspace overview thumbnail placement.

**Comment by mikehardy** [comment]
Oops! Yes, I meant workspace-matrix.

**Comment by cyberhck** [comment]
I agree with @mikehardy here, I just tried workspace matrix, and it's working good with latest version as well, I think it's good to start creating issues on what workspace matrix can take from this project,

then finally deprecate in favor of workspace matrix :)
